forte/data/data_pack.py

```diff
@@ -171,7 +170,7 @@ def __init__(self, pack_name: Optional[str] = None):
         self._data_store: DataStore = DataStore()
         self._entry_converter: EntryConverter = EntryConverter()
         self.image_annotations: List[ImageAnnotation] = []
-        self.grids: List[Grids] = []
+        self.grid: Optional[Grid] = None
```

Comment:
This actually not used in test cases and seems like it's not used in #859 or any other CV-related PRs. Is there any specific plan on how it will be used later?